srv/apns/push_service.go

```diff
@@ -82,38 +84,81 @@ func (p *pushService) BuildPushServiceProviderFromMap(kv map[string]string, psp
 	}
 
 	if cert, ok := kv["cert"]; ok && len(cert) > 0 {
+		// Binary API
 		psp.FixedData["cert"] = cert
```

Comment:
would prefer 2 helper functions/methods returning `err` for the contents of if/else